source/collections/Vector2D.ooc

```diff
@@ -38,6 +38,52 @@ Vector2D: class <T> {
 	_elementPosition: func (row, column: Int, columnCount := this columnCapacity) -> Int {
 		result := columnCount * row + column
 	}
+	resize: func (newRowCapacity, newColumnCapacity: Int) {
+		temporaryResult: T*
+
+		maximumRowCapacity := this rowCapacity > newRowCapacity ? newRowCapacity : this rowCapacity
+		maximumColumnCapacity := this columnCapacity > newColumnCapacity ? newColumnCapacity : this columnCapacity
+
+		if (newRowCapacity == this rowCapacity && newColumnCapacity == this columnCapacity)
+			return
+
+		temporaryResult = gc_calloc(newRowCapacity * newColumnCapacity, T size)
```

Comment:
True. On the same note, if both newRowCapacity and newColumnCapacity are smaller than rowCapacity and columnCapacity respectively one could use malloc instead of calloc since the zero-initialization won't be needed. Should be slightly faster.